<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Guide</class>
 <widget class="QDialog" name="Guide">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>451</width>
     <height>441</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>가이드</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>가능상태</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>661</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>가이드 현황</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>60</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>610</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>돌아가기</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>451</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>580</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>580</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>가이드를 선택하였습니다.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
